web-common/src/test/java/edu/unc/lib/boxc/web/common/services/AccessCopiesServiceTest.java

```diff
@@ -94,6 +101,22 @@ public void init() throws IOException, SolrServerException {
                 ORIGINAL_FILE.getId() + "|image/png|file.png|png|766|urn:sha1:checksum|");
         mdObjectImg.setDatastream(imgDatastreams);
 
+        mdObjectAudio = new ContentObjectSolrRecord();
+        mdObjectAudio.setId("45c8d1c5-14a1-4ed3-b0c0-6da67fa5f6d1");
+        List<String> audioDatastreams = Collections.singletonList(
+                ORIGINAL_FILE.getId() + "|audio/mpeg|file.mp3|mp3|766|urn:sha1:checksum|");
+        mdObjectAudio.setDatastream(audioDatastreams);
+
+        noOriginalFileObj = new ContentObjectSolrRecord();
+        noOriginalFileObj.setId("45c8d1c5-14a1-4ed3-b0c0-6da67fa5f6d1");
+        hasPermissions(noOriginalFileObj, true);
+
+        mdObjectXml = new ContentObjectSolrRecord();
+        mdObjectXml.setId("45c8d1c5-14a1-4ed3-b0c0-6da67fa5f6d1");
```

Comment:
You should switch to using different ids for each test object, otherwise you won't be able to tell if its generating download URLs to the wrong object. You could use `UUID.randomUUID().toString()` or hard code the values.